contracts/src/systems/config/contracts.cairo

```diff
@@ -150,7 +150,16 @@ trait IPopulationConfig {
 
 #[dojo::interface]
 trait IMercenariesConfig {
-    fn set_mercenaries_config(ref world: IWorldDispatcher, troops: Troops, rewards: Span<(u8, u128)>);
+    fn set_mercenaries_config(
+        ref world: IWorldDispatcher,
+        knights_lower_bound: u64,
```

Comment:
we could make these move compressed no? would this fit in u32 or smaller?

Reply:
hmmm the knights_count, paladin_count and crossbowman_count inside the Troops structs are all u64
Also it would make the bounds max 4,294,967 troops accounting for the 1000 precision needed